langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaClient.java

```diff
@@ -172,6 +172,32 @@ public EmbeddingResponse embed(EmbeddingRequest request) {
         }
     }
 
+    public ModelListResponse modelList() {
+        try {
+            retrofit2.Response<ModelListResponse> retrofitResponse = ollamaApi.modelList().execute();
+            if (retrofitResponse.isSuccessful()) {
+                return retrofitResponse.body();
+            } else {
+                throw toException(retrofitResponse);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public ModelDetailsResponse modelDetails(ModelDetailsRequest modelDetailsRequest) {
```

Comment:
I am wondering: should we expose these 2 new endpoints the same way as in https://github.com/langchain4j/langchain4j/pull/514/files#diff-cf3a590009831299f0335fe9bd4358e38e5c862a531bf6d046a310daa0843266R64 ?

Reply:
Assuming that `MistralAiModels` represents the API management component, I exposed the new endpoints similarly. My only concern is with the use of the `Response` class, given its model-centric documentation. Despite this doubt, I appreciate the concept of having a dedicated class for the management API.

Reply:
And, I forgot to mention, because the client code is in a package-scoped context, requests and responses are also package-scoped. In the linked code from Mistral, requests and responses are package-scoped, but their components, like `MistralAiModelCard`, are public. I believe some clarification is needed here. Should certain parts of the client code be public? In the Mistral example, only a part of the response is used in `MistralAiModels`. However, in `OllamaModels`, all information from `ShowModelInformationResponse` is necessary. To address this, I introduced `OllamaModelInfo`, which is essentially another name for `ShowModelInformationResponse` with public access. I'm not a huge fan of this solution, so I would greatly appreciate some guidance or clarification.